src/main/scala/io/sdkman/changelogs/JavaMigrations.scala

```diff
@@ -47,4 +47,13 @@ class JavaMigrations {
       Version("java", "9.0.4-openjdk", "https://download.java.net/java/GA/jdk9/9.0.4/binaries/openjdk-9.0.4_osx-x64_bin.tar.gz", MacOSX),
       Version("java", "10.0.0-openjdk", "https://download.java.net/java/GA/jdk10/10/binaries/openjdk-10_osx-x64_bin.tar.gz", MacOSX))
   }
+
+  @ChangeSet(order = "007", id = "007-add_zulu_7_0_171", author = "vpavic")
```

Comment:
As part of this PR can we also clean up the residual 7.0.141-zulu versions please?